Tests/TestingTests/IssueTests.swift

```diff
@@ -1479,4 +1483,18 @@ struct IssueCodingTests {
     #expect(issueSnapshot.sourceLocation == updatedSourceLocation)
     #expect(issueSnapshot.sourceContext.sourceLocation == updatedSourceLocation)
   }
+
+  @Test("Custom descriptions are the same",
+    arguments: issueKinds
+  )
+  func customDescription(issueKind: Issue.Kind) async throws {
+    let issue = Issue(
+      kind: issueKind,
+      comments: ["Comment"],
+      sourceContext: SourceContext(backtrace: Backtrace.current(), sourceLocation: SourceLocation())
+    )
+    let issueSnapshot = Issue.Snapshot(snapshotting: issue)
+
+    #expect(String(describing: issueSnapshot) == String(describing: issue))
```

Comment:
Please also test `String(reflecting:)` so we get code coverage of `debugDescription`.